stdlib/stack_trace.mli

```diff
@@ -0,0 +1,141 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Stack traces.
+
+    Facilities for getting the current stack trace, printing and inspecting
+    them.
+
+    @since 4.08 *)
+
+type t
+(** The type of stack traces.
+
+    Stack traces cannot be marshalled. If you need marshalling, you should use
+    the array returned by the {!slots} function. *)
+
+val current: int -> t
+(** [current n] returns a description of the stack trace on the current program
+    point (for the current thread), with at most [n] entries. *)
+
+val to_string: t -> string
+(** Return a string listing the program locations contained in the stack
+    trace. *)
+
+val length: t -> int
+(** [length st] returns the number of (non-inlined) slots in the stack trace
+    [st]. *)
```

Comment:
Why would this only returned the number of non-inlined frames?  Seems a bit odd.